package.json

```diff
@@ -20,6 +20,7 @@
     "@changesets/cli": "^2.27.1",
     "del-cli": "^5.1.0",
     "prettier": "^3.2.5",
-    "typescript": "^5.4.5"
+    "typescript": "^5.4.5",
+    "undici": "^6.14.1"
```

Comment:
We’re only using `undici` in a single test in `openapi-fetch`, so we want to add it to devDependencies in that `package.json`; not for the entire monorepo.

Reply:
Fixed.